environment.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = { simple_transportation_pack: { intelligence: [:cellphone], items: %i[medipack chevy_versa] },
+                 standard_transportation_pack: { intelligence: %i[cellphone antenna],
+                                                 arsenal: [:colt_1911], items: %i[handcuffs medipack chemistry chevy_versa] },
+                 simple_mission_pack: { intelligence: %i[infopack laptop cellphone antenna],
+                                        arsenal: [:colt_1911], items: %i[handcuffs medipack chemistry financial] },
+                 standard_mission_pack: { intelligence: %i[infopack laptop cellphone antenna],
+                                          arsenal: %i[remington_870 colt_1911 machete hatchet],
+                                          items: %i[handcuffs medipack chemistry financial] } }
+    end
+  end
+
+  class Control
+    attr_accessor :missions
+
+    states = %w[paused aborted failed accomplished]
+
```

Comment:
I would avoid this extra empty line as `states` is stated and used right away.

Reply:
So, is it better to do something like %w[paused aborted failed accomplished].each directly?